src/services/external/elasticsearch/types/contribution.type.ts

```diff
@@ -0,0 +1,21 @@
+const CONTRIBUTION_TYPE = {
+  HUB_JOINED: 'HUB_JOINED',
+  HUB_CONTENT_EDITED: 'HUB_CONTENT_EDITED',
+  CHALLENGE_CREATED: 'CHALLENGE_CREATED',
+  CHALLENGE_CONTENT_EDITED: 'CHALLENGE_CONTENT_EDITED',
+  CHALLENGE_JOINED: 'CHALLENGE_JOINED',
+  OPPORTUNITY_CREATED: 'OPPORTUNITY_CREATED',
+  OPPORTUNITY_CONTENT_EDITED: 'OPPORTUNITY_CONTENT_EDITED',
+  OPPORTUNITY_JOINED: 'OPPORTUNITY_JOINED',
+  CALLOUT_CREATED: 'CALLOUT_CREATED',
+  CALLOUT_COMMENT_CREATED: 'CALLOUT_COMMENT_CREATED',
+  CALLOUT_CARD_CREATED: 'CALLOUT_CARD_CREATED',
+  CALLOUT_CANVAS_CREATED: 'CALLOUT_CANVAS_CREATED',
+  CALLOUT_CARD_COMMENT_CREATED: 'CALLOUT_CARD_COMMENT_CREATED',
+  CALLOUT_CANVAS_EDITED: 'CALLOUT_CANVAS_EDITED',
+  UPDATE_CREATED: 'UPDATE_CREATED',
+} as const;
```

Comment:
Do you need that cast?

Reply:
In this specific use case probably not - but generally I don't want `CONTRIBUTION_TYPE` to be changed further.